src/JSInterop/Microsoft.JSInterop/src/DotNetObjectRefOfT.cs

```diff
@@ -22,7 +22,9 @@ public sealed class DotNetObjectRef<TValue> : IDotNetObjectRef, IDisposable wher
         /// This API is for meant for JSON deserialization and should not be used by user code.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
+#pragma warning disable CS8618 // Non-nullable field is uninitialized.
         public DotNetObjectRef()
```

Comment:
The compiler complains that `Value` is uninitialized here. I don't want to make it null since it's expected not to be in the usual case users are interacting with it.